package.json

```diff
@@ -19,7 +19,8 @@
     "lint:js": "NODE_ENV=test eslint . --fix",
     "format": "prettier --write 'src/**/*.{css,scss,json,html,yaml,md,mdx,js}'",
     "format:check": "prettier --check 'src/**/*.{css,scss,json,html,yaml,md,mdx,js}'",
-    "update-browserslist": "npx browserslist-ga"
+    "update-browserslist": "npx browserslist-ga",
+    "postinstall": "husky"
```

Comment:
The [latest release](https://github.com/typicode/husky/releases/tag/v9.0.1) suggests this should be on `prepare` instead of `postinstall`, what do you think?

Reply:
Ah ok cool. I actually had it `prepare` at first but wanted to be consistent with what was being used elsewhere. But I'll change it 👍